pkg/volume/volume_linux.go

```diff
@@ -33,20 +33,36 @@ const (
 	execMask = os.FileMode(0110)
 )
 
+// calculateMask starts with a read+write mask and opens them further as needed.
+func calculateMask(mounter Mounter, mode os.FileMode, dir bool) os.FileMode {
```

Comment:
Do we have unit test coverage?

Reply:
thought about that, a little afraid the tests would be pretty trivial, any particular case seem interesting?